src/widgets/view/view.rs

```diff
@@ -238,9 +250,19 @@ impl View {
         self.tab_size - (position % self.tab_size)
     }
 
-    fn render_line_str(&self, line: &Line, lineno: usize, styles: &HashMap<u64, Style>) -> String {
+    fn render_line_str(&self, line: &Line, line_no: Option<u64>, lineno: usize, styles: &HashMap<u64, Style>) -> String {
```

Comment:
I think it might get confusing to have both `line_no` and `lineno` arguments. It might make sense to rename one (or possibly both). For example, maybe change `lineno` to `screen_row`, `screen_line`, or `screen_y` to make it clearer that it's just describing the screen coordinates, rather than the position within the file itself.

Reply:
Your right that could be confusing, I'm thinking line_no for the actual file line number and line_index for lineno, what do you think?